inc/inventorycomputerlib.class.php

```diff
@@ -1365,12 +1369,17 @@ function updateComputer($a_computerinventory, $computers_id, $no_history, $setdy
 
 
       // * Batteries
-         /* Standby, see ticket http://forge.fusioninventory.org/issues/1907
+      if ($pfConfig->getValue("component_battery") != 0) {
          $db_batteries = array();
          if ($no_history === FALSE) {
-            $query = "SELECT `id`, `name`, `serial`
-                  FROM `glpi_plugin_fusioninventory_inventorycomputerbatteries`
-               WHERE `computers_id` = '$computers_id'";
+            $query = "SELECT `glpi_items_devicebatteries`.`id`, `designation`, `voltage`, `capacity`,
+                        `devicebatterytypes_id`, `manufacturers_id`
+                        FROM `glpi_items_devicebatteries`
```

Comment:
This is not used. In facts, designation, type and manufacturer are not used; I've dropped them